src/Core/src/Hosting/IAppHost.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Maui.Hosting
+{
+	public interface IAppHost : IHost
+	{
+		IMauiHandlersServiceProvider Handlers { get; }
```

Comment:
Using a new "MAUI" type instead of the base `IHost` to make is much better for consumption. And also matches the changes for the `IAppHostBuilder`